src/cfml/system/modules_app/server-commands/commands/server/prune.cfc

```diff
@@ -0,0 +1,159 @@
+/**
+ * Prune.
```

Comment:
Needs docs.  Remember to write the first sentence such that the words "This command will..." would fit in front of your sentence.  i.e  _"Forget old servers which haven't been started in a given period of time"_.